__tests__/commons/Loading.test.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
```

Comment:
fix typo docter-detail to doctors-detail

Reply:
or you can make it look like the pages folder by writing your test in doctors/[id].tsx

Reply:
ok should i put it in the pages/doctors folder

Reply:
not pages, but __tests__/doctors/[id].tsx